src/main/scala/api.scala

```diff
@@ -201,14 +201,37 @@ case object api {
       ungapped    :&: □
     )
 
-    import ohnosequences.blast.api.outputFields.{qseqid, sseqid}
-    type OutputFields = qseqid :&: sseqid :&: □
-    val outputFields: OutputFields = qseqid :&: sseqid :&: □
+    import ohnosequences.blast.api.outputFields._
+    type OutputFields =
```

Comment:
@rtobes @marina-manrique could you tell if any essential/useful blastn output field is missing here?